sam/models/spc_model.py

```diff
@@ -175,50 +164,20 @@ class SPCRegressor(BaseTimeseriesRegressor):
 
     def __init__(
         self,
-        predict_ahead: int = (1,),
+        predict_ahead: Sequence[int] = (0,),
         quantiles: Sequence[float] = (),
-        use_y_as_feature: bool = False,
         use_diff_of_y: bool = False,
         timecol: str = None,
         y_scaler: TransformerMixin = None,
```

Comment:
In what scenario would you like to do that?

On the other hand the MLP also adds parameters. If we allow `feature_engineering` arguments for SPC, it might suggest that it is also used.

Reply:
Right now for Aquasuite we use the baseclass (BaseTimeSeriesRegressor) as the type hint in the model, so the Aquasuite code assumes that they all share the same init, so we can safely swap the actual implementation when not enough data is present (or if the model performs very badly). That means that the init is exactly the same so Aquasuite does not need to know the exact model that is being used. Right now that would fail and Aquasuite would need to check the model first to determine the init parameters, which is not very clean

Reply:
https://github.com/RoyalHaskoningDHV/sam/issues/40